index.html

```diff
@@ -58,8 +57,207 @@ <h3>I am a Software Developer</h3>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+      <div id="simple-post">
+        <div id="sp-content">
+          <div id="sp-h-wrapper">
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+      <div id="simple-post">
+        <div id="sp-content">
+          <div id="sp-h-wrapper">
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+      <div id="simple-post">
+        <div id="sp-content">
+          <div id="sp-h-wrapper">
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+      <div id="simple-post">
+        <div id="sp-content">
+          <div id="sp-h-wrapper">
+            <h3>Art Printing Data</h3>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul id="sp-lang-wrapper">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <a id="sp-link" href="#">
+          <button id="sp-button">See Project</button>
+        </a>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+<section id="about-me">
+  <div id="am-content">
+    <h2 id="am-heading">About me</h2>
+    <p>
+      Hello, I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+    <a id="am-link" href="#">
```

Comment:
- [optional] kindly practice using classes as well and id for particular sections as id is used for single elements that appear on the page for only once (e.g. header, footer, menu), whereas class is used for single or multiple elements that appear on the page for once or more than once (e.g. paragraphs, links etc)